include/libtorrent/settings_pack.hpp

```diff
@@ -1503,7 +1515,9 @@ namespace libtorrent
 			// bytes down) or the seed time ratio (seconds as seed / seconds as
 			// downloader) or the seed time limit (seconds as seed) it is
 			// considered done, and it will leave room for other torrents these
-			// are specified as percentages
+			// are specified as percentages. They does not disable the torrent,
+			// they do only controll the queue position which may result into an
```

Comment:
I don't think this should be described as a "queue" position. queue position is already a term used for the order in which downloading torrents are started. This affects the seed rank. This sentence should probably be phrased similar to the one above, for seed_time_limit